src/apps/competitions/tests/test_submissions.py

```diff
@@ -246,27 +246,61 @@ class FactSheetTests(SubmissionTestCase):
     def setUp(self):
         super().setUp()
         self.competition.fact_sheet = {
-            "boolean": [True, False],
-            "selection": ["value1", "value2", "value3", "value4"],
-            "text": "",
+            "boolean": {
+                "key": "boolean",
+                "type": "checkbox",
+                "title": "boolean",
+                "selection": [True, False],
+                "is_required": "false",
+                "is_on_leaderboard": "false"
+            },
+            "text": {
+                "key": "text",
+                "type": "text",
+                "title": "text",
+                "selection": "",
+                "is_required": "false",
+                "is_on_leaderboard": "false"
+            },
+            "text_required": {
+                "key": "text_required",
+                "type": "text",
+                "title": "text",
+                "selection": "",
+                "is_required": "true",
+                "is_on_leaderboard": "false"
+            },
+            "selection": {
+                "key": "select",
+                "type": "select",
+                "title": "selection",
+                "selection": ["", "v1", "v2", "v3"],
+                "is_required": "false",
+                "is_on_leaderboard": "true"
+            }
         }
         self.competition.save()
 
     def test_fact_sheet_valid(self):
         submission = SubmissionCreationSerializer(super().make_submission()).data
+        from pprint import pprint
+        pprint(submission)
         submission['fact_sheet_answers'] = {
             "boolean": True,
-            "selection": "value3",
-            "text": "accept any",
+            "selection": "v3",
+            "text_required": "accept_text",
+            "text": "",
         }
-        serializer = SubmissionCreationSerializer(data=submission, instance="PATCH")
+        serializer = SubmissionCreationSerializer(instance='PATCH', data=submission)
```

Comment:
what is this attempting to do? instance should be an instance of a model?

Reply:
I'm not a 100%. I got it from a stack overflow somewhere. If instance='PATCH' or Submission the tests work as expected. If instance isn't defined, then the tests fail.

Reply:
that's... weird, what happens if you pass `instance='ABC'`?

Reply:
That works as well. If I don't have anything I get this error:
![image](https://user-images.githubusercontent.com/16579931/98582269-98815f00-2277-11eb-9458-f29b2bc705e9.png)

Reply:
I think instance should be a model instance, passing a string here definitely seems wrong..